test/unit/math/mix/fun/trace_gen_quad_form_test.cpp

```diff
@@ -19,8 +19,8 @@ TEST(mathMixMatFun, traceGenQuadForm) {
   stan::test::expect_ad(f, c11, a11, b11);
 
   stan::test::ad_tolerances tols;
-  tols.hessian_hessian_ = 1e-1;
-  tols.hessian_fvar_hessian_ = 1e-1;
+  tols.hessian_hessian_ = 0.15;
+  tols.hessian_fvar_hessian_ = 0.15;
```

Comment:
I fixed a bug that prevented all-double overload of `trace_gen_quad_form()` to be ever chosen. While faster, that overload seems to be a bit less accurate, so I had to increase these tolerances.